### DISCUSSION TITLE
Generic Key IMemoryCache

### DISCUSSION THREAD

**Discussion by rambodghx** [dotnet/runtime#88080] [Q&A] [answered]
Hi everyone,
meanwhile I'm trying to work with `IMemoryCache` I faced with situation that my keys are int type.
this is _questionable_ for me why we have not Generic Key implementation of `IMemoryCache` in order to prevent from **boxing** in Get and Set operations.
Hope to deliver correct approach to community.

**Comment by davidfowl** [comment, marked as answer]
You can read more here https://github.com/dotnet/runtime/issues/48567

**Reply by rambodghx** [reply]
Tnx
